src/component/Button/index.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import styled from "styled-components";
+
+const StyledButton = styled.button`
+    padding: ${(props) => (props.paddingH ? props.paddingH : "6px")}
+        ${(props) => (props.paddingV ? props.paddingV : "12px")};
+    margin: ${(props) => (props.margin ? props.margin : "2px")};
+    height: ${(props) => (props.height ? props.height : "35px")};
+    width: ${(props) => (props.width ? props.width : "none")};
+    border: 1px solid
+        ${(props) => (props.borderColor ? props.borderColor : "black")};
+    border-radius: ${(props) => (props.radiusV ? props.radiusV : "5px")}
+        ${(props) => (props.radiusH ? props.radiusH : "5px")};
+    font-size: 14px;
+    background-color: ${(props) =>
+        props.backgroundColor ? props.backgroundColor : "white"};
+    color: ${(props) => (props.textColor ? props.textColor : "black")};
+`;
+
+const Button = (props) => {
+    return (
+        <StyledButton
+            margin={props.margin}
+            textAlign={props.textAlign}
+            borderColor={props.borderColor}
+            backgroundColor={props.backgroundColor}
+            textColor={props.textColor}
+            height={props.height}
+            width={props.width}
+            paddingH={props.paddingH}
+            paddingV={props.paddingV}
+            radiusH={props.radiusH}
+            radiusV={props.radiusV}
+        >
+            {props.children}
+        </StyledButton>
+    );
+};
+
+export default Button;
```

Comment:
`Button` 컴포넌트는 현재 props 를 받아 그대로 `StyledButton` 에 전달하고 있는 역할만을 가지고 있는데요,

```suggestion
const Button = styled.button`
    padding: ${(props) => (props.paddingH ? props.paddingH : "6px")}
        ${(props) => (props.paddingV ? props.paddingV : "12px")};
    margin: ${(props) => (props.margin ? props.margin : "2px")};
    height: ${(props) => (props.height ? props.height : "35px")};
    width: ${(props) => (props.width ? props.width : "none")};
    border: 1px solid
        ${(props) => (props.borderColor ? props.borderColor : "black")};
    border-radius: ${(props) => (props.radiusV ? props.radiusV : "5px")}
        ${(props) => (props.radiusH ? props.radiusH : "5px")};
    font-size: 14px;
    background-color: ${(props) =>
        props.backgroundColor ? props.backgroundColor : "white"};
    color: ${(props) => (props.textColor ? props.textColor : "black")};
`;

export default Button;
```

위와 같이 수정하면 더 좋은 코드가 될 것 같아요. 😄

Reply:
매개변수 선언 부분이 없어도 argument를 받아올 수 있다는 것을 처음 알았습니다.. 단순한 역할만 하는 코드면 그냥 css처럼 만들면 되는 거였군요!!
<img width="610" alt="image" src="https://user-images.githubusercontent.com/64801796/168457634-9a831393-5ddb-4fbd-8018-bea50e4dbae3.png">
수정 완료 했습니다!